lib/l10n/app_en.arb

```diff
@@ -951,5 +954,291 @@
+      "typeName": {
+        "type": "String"
+      },
+      "itemCount": {
+        "type": "int"
+      }
+    }
+  },
+  "downloadLibraryPrompt": "Are you sure you want to download all contents of the library '{ libraryName }'?",
+  "@downloadLibraryPrompt": {
+    "description": "Confirmation prompt before downloading entire library, which may be large.",
+    "placeholders": {
+      "libraryName": {
+        "type": "String"
+      }
+    }
+  },
+  "onlyShowFullyDownloaded": "Only show fully downloaded albums",
+  "@onlyShowFullyDownloaded": {
+    "description": "Tooltip for button that hides all partially downloaded albums"
+  },
+  "filesystemFull": "Remaining downloads cannot be completed.  The filesystem is full.",
+  "@filesystemFull": {
+    "description": "Message shown when a download cannot complete due to the filesystem being full."
+  },
+  "connectionInterrupted": "Connection interrupted, pausing downloads.",
+  "@connectionInterrupted": {
+    "description": "Message shown when the connection is interrupted during a download."
+  },
+  "connectionInterruptedBackground": "Connection was interrupted while downloading in the background.  This can be caused by OS settings.",
+  "@connectionInterruptedBackground": {
+    "description": "Message shown when the connection is interrupted during a download while in the background.  It advises checking OS settings, as they may cause background downloads to be blocked."
+  },
+  "connectionInterruptedBackgroundAndroid": "Connection was interrupted while downloading in the background.  This can be caused by enabling 'Enter Low-Priority State on Pause' or OS settings.",
+  "@connectionInterruptedBackgroundAndroid": {
+    "description": "Message shown when the connection is interrupted during a download while in the background on android.  It advises checking OS settings, as they may cause background downloads to be blocked, or checking the  'Enter Low-Priority State on Pause' setting."
+  },
+  "activeDownloadSize": "Downloading...",
+  "@activeDownloadSize": {
+    "description": "Message shown for download size while item is still being downloaded"
+  },
+  "missingDownloadSize": "Deleting...",
+  "@missingDownloadSize": {
+    "description": "Message shown for download size while item is still being downloaded"
+  },
+  "syncingDownloadSize": "Syncing...",
+  "@syncingDownloadSize": {
+    "description": "Message shown for download size while item is still syncing."
+  },
+  "runRepairWarning": "The server could not be contacted to finalize downloads migration.  Please run 'Repair Downloads' from the downloads screen as soon as you are back online.",
+  "@runRepairWarning": {
+    "description": "Message shown when an error occurs during the downloads repair step of the Isar downloads migration, which requires a server connection."
+  },
+  "downloadSettings": "Download Settings",
+  "@downloadSettings": {},
+  "showPlaylistSongs": "Show Items with Unknown Library.",
```

Comment:
the token name is a bit unclear. And instead of "Unknown Library", maybe use "Show non-libary items"

Reply:
Yeah, the token name is out of date.  With the library retrieval fixes, I think downloading albumless songs via a playlist is the only way to get items affected by this setting.  Is non-library a good term?  They probably belong to a library, we just don't know which.

Reply:
> I think downloading albumless songs via a playlist is the only way to get items affected by this setting

Well there is a `Playlists` library/collection/parent in Jellyfin that doesn't show up as a regular library. This was actually a [bug in 10.8.11](https://github.com/jellyfin/jellyfin/issues/10272). So there are "global" playlists and user playlists, which are not the same. The "Playlists" library doesn't even show up in the admin dashboard when giving users access to specific libraries, but it does show up when you give them access to *all* libraries. It's weird.  
There's a monkey-patch in 10.8, and afaik they're not sure how they want to handle playlists going forward in 10.9.

Reply:
That just means regular playlists are non-library items, so calling this setting that would be confusing because it doesn't affect them, only songs downloaded via them that can't be assigned correctly.

Reply:
Ah, so the playlists themselves work, but the songs inside of them wouldn't be shown?  
How about "Show songs from global playlists" then?

Reply:
As a hopefully clearer explanation, this setting is for showing download items with a null viewID field.  Playlist will always have this field be null and will be shown in every library regardless of this setting.  Songs downloaded via playlist initially come with a null view id, but we attempt to look them up by album and find the appropriate library to assign them too.  If that fails, this field remains null and their visibilility outside of their playlist is controlled by this setting.  If true, these songs, as well as the associated artists/genres, will be shown in all libraries.  Otherwise they will be hidden in all librarties and only accessible via the playlist that downloaded them.  This setting was a lot more meaningful when I didn't have a way to look up what library any of the songs in a playlist were supposed to be in.

Reply:
Right, I dimly remember. The issue was that songs were downloaded, but didn't show up on the songs tab...  
In that case, keep this as it is.

Reply:
Here too, please rename to "media" or something similar